phpBB/styles/prosilver/template/overall_header.html

```diff
@@ -28,19 +28,19 @@
 -->
 
 <!-- IF S_ALLOW_CDN -->
-<script>
+<script type="text/javascript">
 	WebFontConfig = {
-		google: {
-			families: ['Open Sans:n6']
-		}
+		google: { families: [ 'Open+Sans:600:cyrillic-ext,latin,greek-ext,greek,vietnamese,latin-ext,cyrillic' ] }
 	};
-
-	(function(d) {
-		var wf = d.createElement('script'), s = d.scripts[0];
-		wf.src = 'https://ajax.googleapis.com/ajax/libs/webfont/1.5.18/webfont.js';
-		wf.async = true;
+	(function() {
```

Comment:
Please undo the changes to this part. We decided to have them this way for a reason. Please only change the families setting above.